doc/examples/setup_assistant/setup_assistant_tutorial.rst

```diff
@@ -1,383 +1,524 @@
+* Set the child link as ``panda_link0`` and the parent frame name as ``world``.
 
-* Set the child link as "panda_link0" and the parent frame name as "world".
+* Set the Joint Type as ``fixed``.
 
-* Set the Joint Type as "fixed".
-
-* Click *Save* and you should see this screen:
+* Click **Save** and you should see this screen:
 
 .. image:: setup_assistant_panda_virtual_joints.png
    :width: 700px
+   :align: center
+
+Note: Virtual joints are particularly beneficial for robots with mobile bases,
+such as mobile manipulators. They enable modeling the motion of the robot's base,
+which is essential for motion planning and control. For instance, a virtual planar
+joint can be used to connect the robot base frame to the odometry frame,
+effectively representing the robot's movement in the environment.
 
 Step 4: Add Planning Groups
 ---------------------------
 
-Planning groups are used for semantically describing different parts
-of your robot, such as defining what an arm is, or an end effector.
+Planning groups in MoveIt semantically describe different parts of the robot,
+such as the arm or end effector, to facilitate motion planning.
+
+A move group can be configured to correspond to a specific kinematic chain on the robot,
+which is a set of links and joints that define a sequence of transformations
+from the base of the robot to the end effector. For example, a move group might be defined to
+represent the arm of a robot, which would consist of all the links and joints necessary to move the arm.
 
-* Click on the *Planning Groups* pane selector.
+Move groups can also be represented by sets of links or joints corresponding on the robot.
+For example, a move group might be defined to represent the gripper of a robot,
+which would consist of all the links or joints necessary to move together to realize a gripper opening motion.
 
-* Click on *Add Group* and you should see the following screen:
+* Click on the **Planning Groups** pane selector.
 
-.. image:: setup_assistant_panda_planning_groups.png
+* Click on **Add Group** and you should see the following screen:
+
+.. image:: planning_groups/setup_assistant_panda_planning_groups.png
    :width: 700px
+   :align: center
 
-Add the arm
+Add the arm group
 
 * We will first add Panda arm as a planning group
 
-  * Enter *Group Name* as **panda_arm**
+  * Enter **Group Name** as ``panda_arm``.
 
-  * Choose *kdl_kinematics_plugin/KDLKinematicsPlugin* as the
-    kinematics solver. *Note: if you have a custom robot and would
-    like a powerful custom IK solver, see* :doc:`Kinematics/IKFast </doc/examples/ikfast/ikfast_tutorial>`
+  * Choose **kdl_kinematics_plugin/KDLKinematicsPlugin** as the
+    kinematics solver. Note: if you have a custom robot and would
+    like a powerful custom IK solver, see :doc:`IKFast Kinematics SolverIKFast </doc/examples/ikfast/ikfast_tutorial>` and :doc:`TRAC-IK Kinematics Solver </doc/examples/trac_ik/trac_ik_tutorial>`.
```

Comment:
I'd reword this to be a little less tied to custom robots or "power" of an IK solver.

Also, `trac_ik` isn't yet available in ROS 2... but that will change soon if all goes well :)

```suggestion
  * Choose **kdl_kinematics_plugin/KDLKinematicsPlugin** as the kinematics solver, as this is the default for MoveIt.
    Alternatively, you can use other plugins such as :doc:`IKFast </doc/examples/ikfast/ikfast_tutorial>` or `pick_ik <https://github.com/PickNikRobotics/pick_ik>`_.
```